tcms/issuetracker/base.py

```diff
@@ -57,7 +57,9 @@ def _report_comment(self, execution):  # pylint: disable=no-self-use
         comment = "Filed from execution %s\n\n" % execution.get_full_url()
         comment += "**Product:**\n%s\n\n" % execution.run.plan.product.name
         comment += "**Component(s):**\n%s\n\n" % \
-                   execution.case.component.values_list('name', flat=True)
+                   ' '.join([str(i) for i in
+                             execution.case.component.values_list('name',
+                                                                  flat=True)])
```

Comment:
make this a `@staticmethod` inside the base class and use it like that instead of duplicating code.